adflow/pyADflow.py

```diff
@@ -4514,15 +4514,15 @@ def _getDefaultOptions():
             # Physics Parameters
             'discretization':[str, ['central plus scalar dissipation', 'central plus matrix dissipation', 'upwind']],
             'coarseDiscretization':[str, ['central plus scalar dissipation', 'central plus matrix dissipation', 'upwind']],
-            'limiter':[str, ['vanalbeda', 'minmod', 'nolimiter']],
-            'smoother':[str, ['runge kutta', 'dadi']],
-            'equationType': [str, ['euler', 'laminar ns', 'rans']],
+            'limiter':[str, ['vanAlbada', 'minmod', 'noLimiter']],
```

Comment:
Since we are already breaking some backwards compatibility, do we want to "fully" update the options? e.g. `van Albada`, `no limiter` and `k-omega`?

Reply:
I'm asking since I see a lot of options already have spaces, like `Runge Kutta` instead of `rungeKutta`.

Reply:
I don't see why not at this point. Most of the string options do use spaces so I think it would be nice to make it consistent. What about dashes? We don't use those anywhere. Is it better to stick with spaces?

Reply:
I think we should use dashes if that's what done in the literature, e.g. `Wilcox k-omega`, but if it's just two words then we should split with spaces. Overall, I think it's fine to keep option names as single words with camelCase, but the values could be expanded to have spaces (or dashes if appropriate). What do you guys think @eirikurj @anily @joanibal

Reply:
I added them for now, but it's easy enough to change back.

Reply:
I agree. For string values we should make them more readable by separating using spaces when appropriate.